### ISSUE TITLE
Updating Word Vector Models

### ISSUE THREAD

**Issue by HarshavardhanG513** [explosion/spaCy#3534]
Is it possible to add a feature to update a pre-trained model, so as to make it more relevant for the user's particular use case? if not what would you suggest i should do. 
Background: i am working on a project, for which i need to be able to match generic words like database programming to specific skillsets such as MySQL knowledge, etc.

**Comment by ines** [comment]
Yes, you can update existing models with more examples to fine-tune them on your data. Can find more details on that in the training documentation:
https://spacy.io/usage/training#ner

Not to be confused with word vectors – if you have custom word vectors for your domain, those can sometimes help boost accuracy. However, this means you'll have to use the same vectors during training and at runtime, so you'll need to train your model from scratch.

You might also want to look into combining the model with rules for specific terminology, e.g. using the `EntityRuler`: https://spacy.io/usage/rule-based-matching#entityruler
